sw/device/tests/crypto/cryptotest/json/sha_sca_commands.h

```diff
@@ -0,0 +1,60 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_JSON_SHA_SCA_COMMANDS_H_
+#define OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_JSON_SHA_SCA_COMMANDS_H_
+#include "sw/device/lib/ujson/ujson_derive.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define SHASCA_CMD_MAX_BATCH_DIGEST_BYTES 128
```

Comment:
Shouldn't this be 256 as we're using 8 words of 32 bits for the digetst? Or do we just transmit the first 4 out of 8 words?

Reply:
Good point, we are actually transmitting 256 bits, i.e., 32 bytes is the correct value.